src/AppInstallerCommonCore/Public/winget/MsixManifest.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+
+#include <AppxPackaging.h>
+#include <AppInstallerStrings.h>
+#include <AppInstallerVersions.h>
+#include <wrl/client.h>
+
+#include <string>
+
+namespace AppInstaller::Msix
+{
+    using namespace Microsoft::WRL;
+    using string_t = Utility::NormalizedString;
+
+    typedef Utility::FourPartsVersionNumber PackageVersion;
```

Comment:
Why not `using PackageVersion = Utility::FourPartsVersionNumber`?

Reply:
Updated